setup.py

```diff
@@ -1,7 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
we should keep this file

Reply:
Would you elaborate on why you would want that? `setup.py` is deprecated, and you should instead use `pip`, `python -m build`, etc. commands instead that are PEP517 compatible.

As a Fedora packager, the only issue that I've encountered is in packaging for EPEL9 due to ancient `setuptools` that do not support PEP621, but that is easily resolved by moving to `hatchling` and this can even be done downstream by the packager only if upstream does not wish to move to `hatchling`